project/Dependencies.scala

```diff
@@ -150,13 +150,13 @@ object Dependencies {
   def excludeSpringBoot = ExclusionRule("org.springframework.boot")
   def excludeSpringAop = ExclusionRule("org.springframework.spring-aop")
   def excludeSpringData = ExclusionRule("org.springframework.data")
-  def excludeSpringFramework = ExclusionRule("org.springframework")
+  // def excludeSpringFramework = ExclusionRule("org.springframework")
```

Comment:
this change was necessary when I was writing the tests...apparently the org.springframework.HttpStatus was needed to throw a NotFoundException